TFNetworkRecLayer.py

```diff
@@ -720,6 +722,15 @@ def is_prev_step_layer(cls, layer):
       return layer.is_prev_time_frame
     return False
 
+  def get_sub_layer(self, layer_name: str):
```

Comment:
This is Python 3 only. We still want to keep Python 2 support (not sure for how long, but for now).